github/repos_projects_test.go

```diff
@@ -416,7 +441,7 @@ func TestRepositoriesService_MoveProjectCard(t *testing.T) {
 		}
 	})
 
-	_, err := client.Repositories.MoveProjectCard("o", "r", 1, input)
+	_, err := client.Repositories.MoveProjectCard(1, input)
```

Comment:
Also, these tests will need to hit the new future `Projects` service, it'll no longer be `client.Repositories`.